app/main/presenters/service_presenters.py

```diff
@@ -29,6 +30,9 @@ def _add_as_attribute_if_key_exists(self, key, service_data):
 
     def __init__(self, service_data, manifest, lots_by_slug, declaration=None):
         self.summary_manifest = manifest.summary(service_data)
+        # get attributes in format suitable for govukSummaryList
+        for section in self.summary_manifest:
+            section.summary_list = to_summary_list_rows(section.questions)
```

Comment:
I think we want to include `preserve_line_breaks` here (but not `format_links`)?

Reply:
Good spot!